taskchampion/taskchampion/src/errors.rs

```diff
@@ -1,9 +1,13 @@
+use std::io;
 use thiserror::Error;
 
-#[derive(Debug, Error, Eq, PartialEq, Clone)]
+#[derive(Debug, Error)]
 #[non_exhaustive]
 /// Errors returned from taskchampion operations
 pub enum Error {
+    /// A crypto-related error
+    #[error("Crypto Error: {0}")]
+    Crypto(String),
```

Comment:
It looks like the places this is used are more generally "server-related", so maybe `Server(..)`?